Dockerfile

```diff
@@ -1,7 +1,13 @@
 FROM debian:bullseye-slim
 
+ARG INCLUDE_DMARC=false
+ENV INCLUDE_DMARC=${INCLUDE_DMARC}
```

Comment:
I think we should default this to 'true'. Otherwise it seems likely someone will forget to set it when building our official image (IIRC we don't have any CI for building/publishing this image)